final_modeling.R

```diff
@@ -826,6 +894,296 @@ rsd_boxplot
+  geom_hline(yintercept = masses$`Start Weight`[3], size = 1, color='black')+
+  # geom_hline(yintercept = (mouse1_compressed$pred_weight_m6[vid_id]), size = 1, color='red')+
+  labs(title = "Mouse 3, M6 framewise prediction (no touch), corrected data", x="Frame", y="Mass (g)")+    
+  facet_wrap(vars(vid_id),ncol=4,scales='free_x',labeller=vid_labeller)+
+  theme_bw(base_size=16)+
+  theme(strip.background = element_blank(),axis.text.x = element_text(angle = 55,vjust=0.75))+
+  scale_x_continuous(n.breaks=15)
+
+
+mouse3_predplot_everyvid
+
+
+
+mouse1_median_mass <- median(concat_mouse1_cleanvids_framewise$pred_weight_m6)
+mouse2_median_mass <- median(concat_mouse2_cleanvids_framewise$pred_weight_m6)
+mouse3_median_mass <- median(concat_mouse3_cleanvids_framewise$pred_weight_m6)
+
+#assuming 30fps
+composite_predplot_everyvid <-  ggplot(mapping = aes(x = frame/30))+
+  geom_hline(yintercept = mouse1_median_mass, size = 0.8, color=composite_pallete[1],linetype="solid")+ #composite_pallete[3])+
+  geom_hline(yintercept = mouse2_median_mass, size = 0.8, color=composite_pallete[2],linetype="solid")+ #composite_pallete[3])+
+  geom_hline(yintercept = mouse3_median_mass, size = 0.8, color=composite_pallete[3],linetype="solid")+ #composite_pallete[3])+
+  geom_line(data=concat_mouse1_cleanvids_framewise, aes(y = pred_weight_m6, color = mouseid), linewidth=0.5)+
+  geom_line(data=concat_mouse2_cleanvids_framewise, aes(y = pred_weight_m6, color = mouseid), linewidth=0.5)+
+  geom_line(data=concat_mouse3_cleanvids_framewise, aes(y = pred_weight_m6, color = mouseid), linewidth=0.5)+
+  geom_hline(yintercept = masses$`Start Weight`[1], size = 0.8, color="black",linetype="dashed")+ #composite_pallete[1])+
+  geom_hline(yintercept = masses$`Start Weight`[2], size = 0.8, color="black",linetype="dashed")+ #composite_pallete[2])+
+  geom_hline(yintercept = masses$`Start Weight`[3], size = 0.8, color="black",linetype="dashed")+ #composite_pallete[3])+  
+  labs(x="Time (s)", y="Observed Mass (g, black), Predicted Mass (g, color)", color="Predicted Mass")+ 
+  # ylim(0,42)+
+  facet_wrap(vars(vid_id),ncol=4,labeller=vid_labeller)+
+  theme_bw(base_size=18)+
+  theme(strip.background = element_blank(),
+        axis.text.x = element_text(size=14,angle = 0,vjust=0.75),
+        strip.text = element_text(size=16,angle = 0,vjust=0.75),
+        panel.grid.minor = element_blank(),
+        panel.grid.major = element_blank(),
+        legend.position="NA")+
+  scale_color_manual(values = composite_pallete,labels=c("Mouse 1", "Mouse 2", "Mouse 3"))+
+  # scale_color_manual(values = pals::kelly()[c(4,6,7)],labels=c("Mouse 1", "Mouse 2", "Mouse 3"))+
+  scale_x_continuous(n.breaks=5) #, labels=scales::label_number(precision = 0))
+
+composite_predplot_everyvid
+
+ggarrange(mouse1_predplot_everyvid,mouse2_predplot_everyvid,mouse3_predplot_everyvid,nrow=3)
+
+
+
+
+
 #################################### Misc stuff, Gautam & Brian code #################################### 
```

Comment:
Is this stuff used? It might be better to just delete it and leave it in the git history if we need it back.